src/modules/events-app.module/module.html

```diff
@@ -4,11 +4,43 @@
 {% set IMAGES = {
   'close': get_asset_url('../../images/close.png'),
   'map': get_asset_url('../../images/map.png'),
-  'city': get_asset_url('../../images/city.jpg'),
-  }
-%}
+  'city': get_asset_url('../../images/city.jpg'), } %}
+
+{%- macro outputColorIfSet(property, color) -%}
+  {%- if color.color -%}
```

Comment:
This is minor, but could the conditional logic live outside the CSS selectors so we don't end up with empty CSS rules if no color has been set (now that the `inherit` else has ben removed)?

Said another way, could the macro also generate the selector, but only if it's needed?